src/app/maintenance/maintenance.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-maintenance',
+  templateUrl: './maintenance.component.html',
+  styleUrls: ['./maintenance.component.css']
+})
+export class MaintenanceComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit() {
+  }
```

Comment:
Just out of interest, do we actually need the empty constructor and ngOnInit?

Reply:
No, this was auto-generated, but I guess I should remove it